HW_3/task16.py

```diff
@@ -0,0 +1,26 @@
+'''Требуется вычислить, сколько раз встречается некоторое число X в массиве A[1..N]. 
+Пользователь в первой строке вводит натуральное число N – количество элементов в массиве. 
+В последующих  строках записаны N целых чисел Ai. Последняя строка содержит число X
+5
+1 2 3 4 5
+3
+1'''
+
+from random import randint
+
+N = int(input("Введите количество элементов: "))
+
+lst = []
+for i in range(N):
+    lst.append(randint(0, 10))
+print(lst)
+
+X = int(input("Введите число: "))
+
+i = 0
+count = 0
+for i in range(len(lst)):
+    if lst[i] == X:
+        count += 1
+        i += 1
```

Comment:
Переменная `i` не нужна. Это Python и циклы `for` в этом языке имеет имеет другую логику. Это просто инструмент перебора.
```suggestion
count = 0
for i in range(N):
    if lst[i] == X:
        count += 1
```